src/widget.rs

```diff
@@ -210,7 +210,8 @@ impl<'a> Clone for WidgetStr<'a> {
 impl<'a> Drop for WidgetStr<'a> {
     fn drop(&mut self) {
         let refcount = self.refcount.get();
-        self.refcount.set(refcount - 1);
+        println!("refcount of '{}' was {}", self.data, refcount);
```

Comment:
As a courtesy to the user, we shouldn't have the library printing stuff unless we're using a debug logger.